prime-router/docs/onboarding-users/transport/rest.md

```diff
@@ -0,0 +1,165 @@
+#### Assumptions
+* First and foremost: This assumes you have credentials with the CDC and have been given access to the Azure portal. If you do not have credentials, and access to the Azure portal, stop here. You cannot go any further.
+* This also assumes you have access to KeyBase, and are part of the prime_dev_ops team there.
+* Also, we assume you have KeyBase installed, and have in mounted into /Volumes/. If you don't have it mounted as a drive, you can just drag and drop the files into KeyBase. If you feel you should have access to the prime_dev_ops team in KeyBase, contact your dev lead, or the dev ops team lead and request access.
+* Finally, this assumes you are working on a Unix-style system such as Linux or MacOS. You can probably do this on Windows as well provided you have a conformant shell with putty and openssl installed. The Git Bash shell would probably do, or the Linux Subsystem for Windows would as well, but no promises are made.
+* Have an active Okta admin account.
+## 1.) How ReportStream REST stransport works
+When STLT message is ready to be sent, ReportStream uploads the message to STLT's ready queue folder found in Azure Blobstorage (e.g., la-phd.elr, ca-doh.elr, etc.) and then triggers the send queue to start the SendFunction.
+<ol type="a">
+    <li>Using receiver.TransportType, the SendFunction calls to specific transport service accordingly. In our case, the SendFunction calls to the RESTTransport service.</li>
+    <li>Reading credential from Azure Vault, RESTTransport uses its Type to determine the authentication type (i.e., UserPass or UserApiKey).  Next, it uses the obtained key to authenticates with STLT and sends the message payload.</li>
+</ol>
+
+## 2.) How to prepare a key for authentication
+For authentication, you need to do both step a and step b.
+<ol type="a">
+<li>Generate the "Credential in JSON format" for authentication</li>
+
+Currently, RESTTransport uses the one of the three options: i) UserPass, ii) UserApiKey with JKS, or iii) UserApiKey with two-legged credential type to authenticate and obtain Bearer token from STLT. User can use **primeCLI** command with credential-create option to generate the "Credential in JSON format" as given below.
```

Comment:
```suggestion
Currently, RESTTransport uses the one of the three options: 
* i) UserPass, 
* ii) UserApiKey with JKS, or 
* iii) UserApiKey with two-legged credential type to authenticate and obtain Bearer token from the STLT. 

User can use **primeCLI** command with credential-create option to generate the "Credential in JSON format" as given below.
```
Separating bullet points into separate lines for readability.

Reply:
Fixed